### ISSUE TITLE
gonic -> maloja listenbrainz api issue

### ISSUE THREAD

**Issue by duckfromdiscord** [krateng/maloja#241]
I pointed [gonic](https://github.com/sentriz/gonic/)'s listenbrainz api to my maloja endpoint and it isn't working, here's the logs on maloja:

```
[debug] Keys: {'listen_type': 'single', 'payload': [{'listened_at': 1694125191, 'track_metadata': {'additional_info': {'track_length': 199}, 'artist_name': '...', 'track_name': '...', 'release_name': ...'}}], 'Host': '127.0.0.1:xxx', 'User-Agent': 'Go-http-client/1.1', 'Content-Length': '303',
 'Authorization': 'Token xxx', 'Content-Type': 'application/json', 'Accept-Encoding': 'gzip'}
[apis] Error with Listenbrainz API: <class 'maloja.apis._exceptions.InvalidMethodException'> (Request: ['submit-listens'])
[debug_performance] Executed jinja_page (('track',), {}) in 0.27s
[apis] Listenbrainz API request: ['1', 'submit-listens']
[debug] Could not find a handler for method 1 in API Listenbrainz
```

The endpoint I provided is `http://127.0.0.1:xxx/apis/listenbrainz/1`

**Comment by duckfromdiscord** [comment]
Using
`http://127.0.0.1:xxx/apis/listenbrainz/` causes:
```
2023/09/07 22:53:17 received bad listenbrainz response:                                             |
HTTP/1.1 404 Not Found                                                                              |
Content-Length: 24                                                                                  |
Content-Type: application/json                                                                      |
Date: Thu, 07 Sep 2023 22:53:17 GMT                                                                 |
Server: waitress                                                                                    |
                                                                                                    |
{"error": "Invalid API"}                                                                            |
2023/09/07 22:53:17 subsonic error code 0: error when submitting:                                   |
>= 400: 404: listenbrainz error
```
on gonic's end (this would be the http response from maloja)

**Comment by duckfromdiscord** [comment]
A temporary fix is to use `http://127.0.0.1:xxx/apis/listenbrainz/1` and modify `apis/_base.py` to ignore the `1` in the path.

**Comment by duckfromdiscord** [comment]
Seems to be right. Will reopen if I have another issue. Thank you 😎
